php/autocomplete.php

```diff
@@ -16,10 +16,12 @@ function trim_query($query) {
 
 header("Content-type: application/json; charset=utf-8");
 
-const SEARCH_TYPES = array("airport", "airline", "plane", "multisearch");
+const SEARCH_TYPES = ["airport", "airline", "plane", "multisearch"];
 
-if (!in_array($_POST['searchType'] ?? null, SEARCH_TYPES)
-    || empty($_POST['searchText'] ?? "")) {
+if (
+    !in_array($_POST['searchType'] ?? null, SEARCH_TYPES)
+    || empty($_POST['searchText'] ?? "")
+) {
```

Comment:
Is there a good PHP auto-formatter we could set up?

Reply:
PHPCS is setup currently, but not enforcing, or run in CI.

I've disabled some rules for now. Have a look at the .phpcs.xml file

You can run 'composer phpcbf' to fix stuff up